plugins/board-resources/src/components/AddCard.svelte

```diff
@@ -0,0 +1,155 @@
+<!--
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+// Copyright © 2021 Hardcore Engineering Inc.
```

Comment:
Only  `Copyright © 2022 Hardcore Engineering Inc.` is required.

Reply:
Done.